doc/quickstart.md

```diff
@@ -100,15 +100,19 @@ We are using J-Link driver in version J-Link v634f ([Ubuntu download](https://ww
 
 **WARNING:** newer J-Link software revisions seem to work fine, but loading via them does not result in usable software on the hardware target.
 
-#### Add GitHub token
+#### Proprietary assets
 
 We are storing assets on github pages and these are downloaded automatically before software images are built.
-Please follow github token configuration here: [download assets documentation](../doc/download_asset.py)
+There are two assets sources - proprietary and open for community.
+
```

Comment:
users still have to configure the token - not configuration of the token is not mentioned explicitly. 

Can you ignore adding the token when you download nonproprietary assets? No, you can not. Therefore it has to be mentioned here.